src/js/pdf-viewer.js

```diff
@@ -6,7 +6,7 @@ angular.module('pdf')
   function($window, $log, pdfDelegate) {
     return {
       restrict: 'E',
-      template: '<pdf-viewer-toolbar delegate-handle="{{id}}" page-count="pageCount"></pdf-viewer-toolbar><canvas></canvas>',
+      template: '<div ng-if="showToolbar"><pdf-viewer-toolbar delegate-handle="{{id}}" page-count="pageCount"></pdf-viewer-toolbar></div><canvas></canvas>',
```

Comment:
@winkerVSbecks You're right. I though that encapsulating pdf-viewer-toolbar into an additional div, the link function wouldn't be executed. But I was plenty wrong.